### PR TITLE
Added verbs/handling for removing and attaching your own robotic bodyparts.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Based on quard's verbs for Neb.

- You can detach non-vital cybernetic limbs with a model that allows it, which are attached to a parent that allows it.
- You can detach non-vital prosthetic limbs with a model that allows it from any kind of parent.
- You can only reattach cybernetic limbs to cybernetic parents that allow it. Prosthetics can reattach to any parent.
- Currently not enabled for any limb, going to revisit it tomorrow and adjust based on feedback from @Schnayy and other mysterious lore cabal people.

~~Tested, appears to work fine - verbs update when your limbs drop off and you can't remove any more.~~ needs significant retesting due to an infestation of feeping creatures.

Additional inclusions after discussion:
- Wooden prosthetics (Morgan Trading Co) ported from Neb.
- Teshari unbranded limbs are a separate company so they can be set to pop off like prosthetics (requested). 

**Current issues:**
- Robotic limbs appear as `blood-stained` or `oil-stained` due to being given blood_DNA on init. Can't fix this without changing some core limb behavior and since it's only cosmetic I'm not sure it's worth potential regressions.
- Detached limbs have no icon in the world, and icon_state is null. I'm not clear on what could be causing this but it's unrelated to the PR so I'm leaving it alone.

### CODE REVIEW

File: code/modules/organs/robolimbs.dm [hunk 1]
```diff
@@ -47,22 +47,24 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	var/icon = 'icons/mob/human_races/robotic.dmi'       // Icon base to draw from.
 	var/monitor_icon = 'icons/mob/monitor_icons.dmi'     // Where it draws the monitor icon from.
 	var/unavailable_at_chargen                           // If set, not available at chargen.
-	var/unavailable_to_build							 // If set, can't be constructed.
-	var/lifelike										 // If set, appears organic.
-	var/skin_tone										 // If set, applies skin tone rather than part color Overrides color.
-	var/skin_color										 // If set, applies skin color rather than part color.
-	var/blood_color = "#030303"
-	var/blood_name = "oil"
+	var/unavailable_to_build                             // If set, can't be constructed.
```
Comment (MistakeNot4892) [inline code/modules/organs/robolimbs.dm:50]: These changes are entirely whitespace normalization other than adding the new `modular_bodyparts` bool.

**Comment by MistakeNot4892** [comment]
Works fine as far as testing shows.

**Comment by Schnayy** [comment]
Apologies if I’m not understanding, it’s 5 am, but would this mean you can’t attach different prosthetic brands onto you?

**Comment by MistakeNot4892** [comment]
To be completely clear:
- This is unrelated to any kind of character generation or surgical modification of bodyparts. That's all untouched. 
- If you take (or have installed) a robolimb with a model that has `modular_bodyparts` set to true, and that bodypart has two or more components of a model with `modular_bodyparts` set (ie. a hand and an arm, or an arm and a chest) you get a verb that allows you to detach the second limb from the first (ie. arm from chest, hand from arm).
- If you have any bodypart that allows for the above, you also get a verb to reattach a `modular_bodypart`-model limb, which checks the above but in reverse - you can plug a hand with a `modular_bodypart` model into an arm with a `modular_bodypart` model.
Sorry this is a bit verbose but trying to write out the logic is a bit noodly.

**Comment by MistakeNot4892** [comment]
DNM again, after discussing on Discord I'm going to try to generalize some of this so it can be used for meat limbs down the track if desired.

**Comment by MistakeNot4892** [comment]
Still need to test this and update the body to mention the flag values but the logic itself is more or less finished. I'll test it and set the vars on the robolimb manufacturers tomorrow after I sleep.

**Comment by Cerebulon** [comment]
Since you asked on discord, brands/models which could probably be easily pop-off-able:
Unbranded (All)
NanoTrasen
Cybersolutions (All)
Einstein Engines
Grayson
Morpheus
Ward-Takahashi (Standard, but NOT Spirit)
Xion (All)

**Comment by MistakeNot4892** [comment]
Still chasing down some bugs with this one but it should be ready for review otherwise.

File: code/modules/organs/robolimbs.dm [hunk 2]
```diff
@@ -47,22 +47,24 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
...
-	var/list/monitor_styles			 		 			 //If empty, the model of limbs offers a head compatible with monitors.
-	var/parts = BP_ALL						 			 //Defines what parts said brand can replace on a body.
-	var/health_hud_intensity = 1						 // Intensity modifier for the health GUI indicator.
-	var/suggested_species = "Human"						 //If it should make the torso a species
-	var/speech_bubble_appearance = "synthetic"			 // What icon_state to use for speech bubbles when talking.  Check talk.dmi for all the icons.
-
-	var/robo_brute_mod = 1								 // Multiplier for incoming brute damage.
-	var/robo_burn_mod = 1								 // As above for burn.
+	// "Species Name" = "Robolimb Company", List, when initialized, will become "Species Name" = RobolimbDatum, used for alternate species sprites.
+	var/list/species_alternates = list(SPECIES_TAJ = "Unbranded - Tajaran", SPECIES_UNATHI = "Unbranded - Unathi")
```
[the remark this answers is not in the document]
Comment (MistakeNot4892) [reply code/modules/organs/robolimbs.dm:67]: I didn't change the code.

**Comment by MistakeNot4892** [comment]
Couple of cosmetic issues mentioned in OP that I haven't nailed down but otherwise this is now feature complete and functional.

**Pull request merged**